mitmproxy/proxy/protocol/http.py

```diff
@@ -327,7 +330,15 @@ def _process_flow(self, f):
                 )
 
                 def get_response():
```

Comment:
Sounds all reasonable and we should fix this, but isn't this a general response streaming issue unrelated to this PR? If so, please open a new issue, and we can address it after merging this PR.